tests/local_clean_urls.php

```diff
@@ -201,6 +201,15 @@ public function test_local_cleanurls_simple() {
         // If we change url config then we need to throw away the cache.
         purge_all_caches();
 
+        $url = 'http://www.example.com/moodle/mod/forum/user.php?id=' . $this->staff->id . '&mode=discussions';
+        $murl = new moodle_url($url);
+        $clean = $murl->out();
+        $this->assertEquals(
+            'http://www.example.com/moodle/user/' . $this->staff->username . '/discussions',
```

Comment:
also I try to avoid as many various in the expected values as possible to make it easier to reason about the process,

ie change this to from

'http://www.example.com/moodle/user/' . $this->staff->username . '/discussions',

'http://www.example.com/moodle/user/head1/discussions',